server/src/routes/api/classes.js

```diff
@@ -17,52 +17,78 @@ var photosUpload = upload.fields([
 
 router.get('/', async (req, res, next) => {
   try {
-    const { hostId } = req.query;
+    const limitValue = parseInt(req.query.limit);
```

Comment:
You are not checking if this query option is even sent? What happens if limit is not sent?

Reply:
The assumption is that the client always sends a value but a check would make sense